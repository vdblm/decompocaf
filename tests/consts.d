int main(){
	Print(0xA2a5);
	Print(0X12f5);
	Print(-0xaaa);
	Print(01234);
	Print(12.);
	Print(12.2E+2);
	Print(12.2E-2);
	Print(12.E+2);
	Print(12.2e+2);
	Print(0012.e+002);
}