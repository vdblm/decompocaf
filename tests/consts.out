41637
4853
-2730
1234
12.00000000
1220.00000000
0.12200000
1200.00000000
1220.00000000
1200.00000000
